cns/endpointmanager/endpointmanager_linux.go

```diff
@@ -0,0 +1,12 @@
+package endpointmanager
+
+import (
+	"context"
+
+	"github.com/Azure/azure-container-networking/cns"
+)
+
+// ReleaseIPs implements an Interface in fsnotify for async delete of the HNS endpoint and IP addresses
+func (em *EndpointManager) ReleaseIPs(_ context.Context, _ cns.IPConfigsRequest) error {
+	return nil
+}
```

Comment:
This is not the correct behavior; it should work if we end up using it. Right now it is noop and won't release IPs. 
It should just call through:
```suggestion
func (em *EndpointManager) ReleaseIPs(ctx context.Context, req cns.IPConfigsRequest) error {
	return em.cli.ReleaseIPs(ctx, req)
}
```